defender/README.md

```diff
@@ -49,4 +49,51 @@ the Autotasks to verify that the Autotasks are performing as expected.
 
 This code uses Jest to override the several modules (`axios`, `ethers`, and `defender-relay-client`) to mock their actions and avoid performing interactions
 with the blockchain or the Internet. This approach allows us to simplify testing by simulating all external data sources and then verifying that Autotasks
-performs the functions we expect.
\ No newline at end of file
+performs the functions we expect.
+
+### Serverless Migration status
+
+NAME - STATUS - (PAUSED)
+
+Production
+- cToken Monitor - Ready to deploy
+- Datadog Alerts Heat Map - Error 400
+- Datadog Forta Bot Alerts - Error 400
+- Datadog Forta Detection Bot Health - Error 400
+- Forta cToken - Ready to deploy - Paused
+- Forta Distribution - Ready to deploy - Paused
+- Forta Explorer Monitor - Ready to deploy but autotask returns error 400, also consider adding more bots and updating old ones
+- Forta Governance - Ready to deploy - Paused
+- Forta Large Borrows Governance - Ready to deploy
+- Forta Large Delegations - Ready to deploy
+- Forta Low Liquidity - Ready to deploy
+- Forta Multi-Sig - Ready to deploy
+- Forta Oracle Price - Ready to deploy
+- Forta Underlying Asset - Ready to deploy
+- Governance Automation - Ready to deploy
+- Governance Discord Alert - Replaces Contract Governance - Ready to deploy
+- Governance Discord Summary - Replaces Active Proposal Updater - Ready to deploy
+- Governance Twitter Bot - Ready to deploy
+- Proposal 110 Automatoooor - Ready to Deploy - Paused
+- Proposal 117 Monitor Market Entered - No longer in production, missing sentinel data - Paused
+- Proposal 117 Monitor Mint - No longer in production, missing sentinel data - Paused
+
+Dev - Yet to be implemented on production
```

Comment:
Add some clarification regarding staged vs downloaded type directories e.g. `<defender-implemenation>_dev contains the most recent code but has yet to be pushed to production`

Reply:
WIll do

Reply:
Addressed in b275551eee16572cc5286da92eadfc4240c2e27b